### DISCUSSION TITLE
YML file can't be imported.

### DISCUSSION THREAD

**Discussion by MaksymTT** [mockoon/mockoon#1220] [General]
Hey all! Have OpenAPI yml file. Wanted to import it via Import/Export menu. When choose import file browsing, .yml files are greyed out, thus not for import (?). But explicitly promised in menu selection "...JSON or YAML". Is it known issue?

**Comment by 255kb** [comment]
Hi, not a bug, rather a feature 😀
The truth is that we never thought about the .yml extension which is not the recommended one, so we only allowed .yaml and .json files in the dialog. 
I will create an issue to add the .yml extension for the next release.
